CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Cromwell Change Log
 
+## 30 Release Notes
+
+### Other changes
+
+* **Database**  
+If desired, the metadata may be optionally stored in a separate SQL database than the of the operations data. See the
+[README](https://github.com/broadinstitute/cromwell#database) or the `database` section in
+[cromwell.examples.conf](https://github.com/broadinstitute/cromwell/blob/develop/cromwell.examples.conf) for more
+information. Cromwell will not move any existing data automatically. When switching connection information for an
```

Comment:
TOL: I would move the "Cromwell will not move any existing data automatically" to be the last sentence in the paragraph, as it is a clarification on the sentence about switching the connection information, and how it will behave.